app/javascript/packages/document-capture/components/in-person-locations.tsx

```diff
@@ -25,15 +25,17 @@ function InPersonLocations({ locations, onSelect, address }: InPersonLocationsPr
   if (locations?.length === 0) {
     return (
       <>
-        <h3>{t('in_person_proofing.body.location.po_search.none_found', { address })}</h3>
+        <h3 role="status">
```

Comment:
It may behave differently depending on specific screen readers, so we could test to check, but my understanding of live regions is that they must be present on the page _before_ adding the content that we want to be announce, since it's "live" in the sense that it announces changes to the content. Typically this would be at page load, but that's impractical for how our React application works.

>The error container must be present in the DOM on page load for the error message to be spoken by most screen readers.

https://www.w3.org/TR/WCAG20-TECHS/ARIA19.html#ARIA19-examples

If testing shows that this works okay as-is, maybe not worth worrying too much over. Otherwise, maybe we adjust the rendering so that the live region is always present (but non-visible) from beginning to end of the searching process, and we add the text once the result is available.

Reply:
This worked as-is on Mac Voiceover, but I didn’t test it with other screen readers.  I’ll do a timeboxed investigation into testing it for other screen readers.

Reply:
Update: I tested on a second screen reader, the Chrome extension screen reader, and the code worked as is.  I am going to merge this.